rslib/src/scheduler/filtered/mod.rs

```diff
@@ -121,7 +121,7 @@ impl Collection {
                 format!("({})", term.search)
             }
         );
-        let order = order_and_limit_for_search(term, ctx.today);
+        let order = order_and_limit_for_search(term, ctx.today, self.storage.creation_stamp()?.0);
```

Comment:
I believe this is only 100% accurate in V1, and we need to get this from the SchedTimingToday calculations in timing.rs instead?

Reply:
Yes, the current approach won't handle the rollover correctly. Please use .days_elapsed instead (eg (due -  {days_elapsed}) * 86400 + {current_timestamp})